sonar-php-plugin/src/test/java/org/sonar/plugins/php/phpunit/xml/TestSuitesTest.java

```diff
@@ -22,115 +22,37 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.Assert.assertSame;
-import static org.junit.Assert.assertThat;
-import static org.junit.matchers.JUnitMatchers.both;
-import static org.junit.matchers.JUnitMatchers.hasItem;
+import static org.junit.Assert.assertTrue;
 
-/**
- * Test for TestSuites.
- *
- * @author gennadiyl
- */
 public class TestSuitesTest {
 
-  private TestSuites testSuites;
-  private static final Logger LOG = LoggerFactory.getLogger(TestSuitesTest.class);
-
-  /**
-   * @throws java.lang.Exception
-   */
-  @Before
-  public void setUp() throws Exception {
-    testSuites = new TestSuites();
-  }
-
-  /**
-   * Test method for {@link org.sonar.plugins.php.phpunit.xml.TestSuites#setTestSuiteList(java.util.List)}.
-   */
-  @Test
-  public void testSetTestSuites() {
-    List<TestSuite> testSuitesList = new ArrayList<TestSuite>();
-
-    testSuites.setTestSuiteList(testSuitesList);
-    assertSame(testSuitesList, testSuites.getTestSuiteList());
-  }
-
-  /**
-   * Test method for {@link org.sonar.plugins.php.phpunit.xml.TestSuites#addTestSuite(org.sonar.plugins.php.phpunit.xml.TestSuite)}.
-   */
-  @Test
-  public void testAddTestSuite() {
-    TestSuite suite = getTestSuite("name");
-
-    testSuites.addTestSuite(suite);
-
-    assertThat(testSuites.getTestSuiteList(), hasItem(suite));
-  }
-
-  /**
-   * Test method for {@link org.sonar.plugins.php.phpunit.xml.TestSuites#addTestSuite(org.sonar.plugins.php.phpunit.xml.TestSuite)}.
-   */
-  @Test
-  public void testAddMultipleTestSuite() {
-    TestSuite suite1 = getTestSuite("name1");
-    TestSuite suite2 = getTestSuite("name2");
-
-    testSuites.addTestSuite(suite1);
-    testSuites.addTestSuite(suite2);
-
-    assertThat(testSuites.getTestSuiteList(), both(hasItem(suite1)).and(hasItem(suite2)));
-  }
-
   /**
    * Empty constructors are required by xstream for TestSuite and TestSuite, in order to
    * be compatible with Java 7.
    */
   @Test
   public void test_compatible_java_7() {
-    Constructor[] constructors1 = getConstructorForClass("org.sonar.plugins.php.phpunit.xml.TestSuite");
-    assertThat(constructors1).isNotEmpty();
-    assertThat(getNbParamPerConstructor(constructors1).first()).isEqualTo(0);
-
-    Constructor[] constructors2 = getConstructorForClass("org.sonar.plugins.php.phpunit.xml.TestSuites");
-    assertThat(constructors2).isNotEmpty();
-    assertThat(getNbParamPerConstructor(constructors2).first()).isEqualTo(0);
-  }
-
-  private Constructor[] getConstructorForClass(String className) {
-    Constructor[] constructors = new Constructor[0];
-    try {
-      Class testSuites = Class.forName(className);
-      constructors = testSuites.getConstructors();
-    } catch (ClassNotFoundException e) {
-      LOG.error("If TestSuite and TestSuites class does not exist any more, this test might be useless.", e);
-    }
-    return constructors;
+    new TestSuites();
+    new TestSuite();
+    assertTrue(true);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove or correct this assertion. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2701)